src/ui/version.coffee

```diff
@@ -15,7 +15,7 @@ check = (noAlert = false)->
         return console.log err if err
         body = JSON.parse body
         tag = body.tag_name
-        releasedVersion = tag.substr(1) # remove first "v" char
```

Comment:
i know but it should not fail unless we have done something wrong with the releases